languages/csharp/reference/exercise-concepts/gigasecond.md

```diff
@@ -1,20 +1,24 @@
-# Concepts of Gigasecond
+# Concepts of gigasecond
 
-[Example implementation](https://exercism.io/tracks/csharp/exercises/gigasecond/solutions)
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/gigasecond/Example.cs)
 
 ## General
 
 - functions: used as the main entry point for the exercise
+- methods: used on the template
+- methods arguments: the time is passed as an argument
```

Comment:
maybe they should be on a  single concept

Reply:
@maurelio1234 ,
I have seen this repeated in almost all the other concept pages.
This separation adds more value to new learners, but for veterans this looks more redundent.
IMO, we should keep it this way.